packages/workflow/src/worker-interface.ts

```diff
@@ -130,7 +130,7 @@ export async function initRuntime({ info, randomnessSeed, now, patches }: Workfl
   // webpack doesn't know what to bundle given a dynamic import expression, so we can't do:
   // state.payloadConverter = (await import(payloadConverterPath)).payloadConverter;
   // @ts-expect-error this is a webpack alias to payloadConverterPath
-  const customPayloadConverter = (await import('__temporal_custom_payload_converter')).payloadConverter;
+  const customPayloadConverter = (await import('__temporal_custom_payload_converter')).payloadConverter; // eslint-disable-line import/no-unresolved
```

Comment:
Or you can use https://github.com/import-js/eslint-plugin-import#importignore instead.